src/App.js

```diff
@@ -12,19 +12,28 @@ import './App.css';
 
 function App() {
   const t = useTranslation();
-  const { authenticated } = useAuth();
+  const { authenticated, user } = useAuth();
+
+  const { email = '' } = user || {};
```

Comment:
WDYT moving this logic to a hook? Let's say `useRootstrapAuth` or something like that?